build/release/release.sh

```diff
@@ -16,6 +16,34 @@ if [[ $1x = x ]]; then
     TAG=$DATE-$VERSION
 else
     TAG=$1
+
+    # If a tag is specified, ensure that that tag is present and checked out.
+    if [[ $TAG != $(git tag -l --points-at HEAD) ]]; then
+        echo "tag $TAG not checked out"
+        exit 1
+    fi
+
+    # Build lnd to extract version.
+    go build github.com/lightningnetwork/lnd/cmd/lnd
+
+    # Extract version command output.
+    LND_VERSION_OUTPUT=`./lnd --version`
+
+    # Use a regex to isolate the version string.
+    if [[ $LND_VERSION_OUTPUT =~ lnd\ version\ (.+)\ commit ]]; then
```

Comment:
To make this more readable and prevent bash version dependency, I'd suggest to change this to:

```
    LND_VERSION_REGEX="lnd version (.+) commit"
    if [[ $LND_VERSION_OUTPUT =~ $LND_VERSION_REGEX ]]; then
```
See https://stackoverflow.com/questions/218156/bash-regex-with-quotes/218217#218217

Reply:
Interesting, didn't know that. Fixed